app/controllers/orders_controller.rb

```diff
@@ -1,24 +1,70 @@
 class OrdersController < ApplicationController
   def cart # One cart (@order) holds information on zero or many OrderItems
-    @order = Order.new(status: "pending")
-    @items = OrderItem.all
+    @order = Order.find_by(status: "pending") # under the assumption there is only one order happening at a time, does this logic work?
+    if @order && @order.order_items.length != 0
+      @items = OrderItem.where(order_id: @order.id)
+    end
   end
 
   def new_order_item
     # TODO: Quantity should come from product#show view; hardcoded for now
-    @item = OrderItem.new(quantity: 1, order_id: @order.id, product_id: params[:id])
+    if Order.find_by(status: "pending")
+      @order = Order.find_by(status: "pending")
```

Comment:
could probably DRY this up but I think this logic makes sense and is great for now!